contribs/gmf/src/directives/mobilegeolocation.js

```diff
@@ -0,0 +1,187 @@
+goog.provide('gmf.MobileGeolocationController');
+goog.provide('gmf.mobileGeolocationDirective');
+
+goog.require('gmf');
+goog.require('ngeo.DecorateGeolocation');
+goog.require('ngeo.FeatureOverlay');
+goog.require('ngeo.FeatureOverlayMgr');
+goog.require('ol.Feature');
+goog.require('ol.Geolocation');
+goog.require('ol.Map');
+goog.require('ol.geom.Point');
+goog.require('ol.proj');
+
+
+/**
+ * Provide a "mobile geolocation" directive.
+ *
+ * @example
+ * <gmf-mobile-geolocation
+ *     gmf-mobile-geolocation-map="ctrl.map"
+ *     gmf-mobile-geolocation-options="ctrl.mobileGeolocationOptions">
+ * </gmf-mobile-geolocation>
+ *
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ */
+gmf.mobileGeolocationDirective = function() {
+  return {
+    restrict: 'E',
+    scope: {
+      'getMobileMapFn': '&gmfMobileGeolocationMap',
+      'getMobileGeolocationOptionsFn': '&gmfMobileGeolocationOptions'
+    },
+    controller: 'GmfMobileGeolocationController',
+    controllerAs: 'ctrl',
+    templateUrl: 'partials/mobilegeolocation.html'
+  };
+};
+
+
+gmfModule.directive('gmfMobileGeolocation', gmf.mobileGeolocationDirective);
+
+
+
+/**
+ * @constructor
+ * @param {angular.Scope} $scope The directive's scope.
+ * @param {ngeo.DecorateGeolocation} ngeoDecorateGeolocation Decorate
+ *     Geolocation service.
+ * @param {ngeo.FeatureOverlayMgr} ngeoFeatureOverlayMgr The ngeo feature
+ *     overlay manager service.
+ * @export
+ * @ngInject
+ */
+gmf.MobileGeolocationController = function($scope, ngeoDecorateGeolocation,
+    ngeoFeatureOverlayMgr) {
+
+  /**
+   * @type {angular.Scope}
+   * @private
+   */
+  this.scope_ = $scope;
```

Comment:
I would expect that we build ngeo in the same conditions as ol3.
Are you sure closure notifies also for unsed properties ?? Or just unused variable ?